.eslintrc.js

```diff
@@ -33,11 +33,18 @@ module.exports = {
     "import/no-named-as-default": "off",
     // "import/no-cycle": [1],
     "no-else-return": "error",
+    "no-console": ["warn", { allow: ["info", "warn", "error"] }],
   },
   reportUnusedDisableDirectives: true,
   overrides: [
     {
-      files: ["**/*.ts?(x)"],
+      files: ["**/*.stories.?(j|t)s?(x)"],
```

Comment:
this glob would actually match files like `thing.stories.s` (often used for assembly code 😂), i'm ok with it.

Reply:
```suggestion
      files: ["**/*.stories.@(j|t)s?(x)"],
```